tests/share/util/test_encrypted_field.py

```diff
@@ -0,0 +1,41 @@
+import pytest
+
+from share.models.fields import EncryptedJSONField
+
+
+class TestEncryptedJsonField:
+
+    @pytest.fixture
+    def field(self):
+        return EncryptedJSONField(null=True, blank=True)
+
+    def test_encrypt_and_decrypt_list(self, field):
```

Comment:
Could you add some tests cases for storing `[]`, `{}`, `set()`, `None`, and `''`

You can use pytest's [parametrize](https://docs.pytest.org/en/latest/parametrize.html) to avoid duplicating a lot of this logic.